Add an action to build an image

parmentf opened the pull request:
Ideally, only when on certain branches

parmentf committed:
feat(terms-teeft): Add service

parmentf committed:
ci(teeft): Add action

parmentf committed:
ci(ws-teeft): Fix context

parmentf committed:
ci(ws-teeft): Maybe with file

See https://github.com/docker/build-push-action/issues/169#issuecomment-
711729397

AlasDiablo commented on .github/workflows/docker-build-on-push.yml:
```diff
@@ -0,0 +1,27 @@
+            - name: Build Docker Image
+              uses: docker/build-push-action@v5
+              with:
+                  context: ./services/terms-teeft
```
```suggestion
                  context: ${{ github.workspace }}/services/terms-teeft
```

AlasDiablo commented on .github/workflows/docker-build-on-push.yml:
```diff
@@ -0,0 +1,27 @@
+              uses: docker/build-push-action@v5
+              with:
+                  context: ./services/terms-teeft
+                  file: ./services/terms-teeft/Dockerfile
```
```suggestion
                  file: ${{ github.workspace }}/services/terms-teeft/Dockerfile
```

parmentf committed:
build(ws-teeft): Use github.workspace

To locate the right Dockerfile

Co-authored-by: AlasDiablo <25723276+AlasDiablo@users.noreply.github.com>

parmentf committed:
build(ws-teeft): Fix absolute path

Error was: unable to prepare context: path
/home/runner/work/actions-labo/actions-labo/services/terms-teeft not
found

parmentf committed:
build(ws-teeft): Let's try more simply
